models/classes/export/preprocessor/PreProcessor.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA;
+ *
+ *
+ */
+
+namespace oat\taoQtiTest\models\export\preprocessor;
+
+
+use qtism\data\storage\xml\XmlDocument;
+
+/**
+ * Interface PreProcessor
+ * Allow to preprocess the test exporter
+ * @package oat\taoQtiTest\models\export\preprocessor
+ */
+interface PreProcessor
```

Comment:
It seems to me it's more an "assessmentItemRefPreProcessor" than a "testPreProcessor". Maybe a more "insightful" name would help.